content/800-guides/1100-using-prisma-nuxt-module-with-prisma-postgres.mdx

```diff
@@ -0,0 +1,267 @@
+   ```prisma file=prisma/schema.prisma
+   datasource db {
+   +  provider = "postgresql"
+   -  provider = "sqlite"
+    url      = env("DATABASE_URL")
+   }
+   ```
+3. Delete the SQLite database files (`dev.db` and `dev.db-journal`) along with the `migrations` folder, all located in the `prisma` directory. This cleans up the existing SQLite setup and prepares your project to migrate to PostgreSQL.
+4. Manually create a new migration for the Postgres database by running the `prisma migrate` command:
+   ```terminal
+   npx prisma migrate dev --name init
+   ```
+5. Start the development server again:
+   ```terminal
+   npm run dev
+   ```
+6. Add a user using Prisma Studio by opening it in the Nuxt DevTools and add a `name` and `email` for a new user.
+   :::note
+   When the server starts, the CLI will display a message explaining how to open Prisma Studio in the Nuxt DevTools:
+    ```terminal
+    After clicking Get Started in Nuxt DevTools, click on the three dots (︙) in the lower left-hand side to reveal additional tabs.
+    Locate the Prisma logo to open Prisma Studio.
+    ```
+   ::: 
+7. Verify the data in the application by refreshing your application at `https://localhost:3000`. The page should display the name of the user you added in Prisma Studio. For example, if you added a user named `Jon`, the application will display `Jon` in the browser.
+
+Congratulations, your Nuxt app is now fully integrated with Prisma Postgres!
+
+## 7. Deploy to Vercel
+
+Deploy your Nuxt application with Prisma Postgres integration to Vercel by following these steps:
+
+1. Ensure your project is version-controlled and pushed to a GitHub repository. If you don’t have a repository yet, [create one on GitHub](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository). Once the repository is ready, run the following commands:
+   ```terminal
+   git add .
+   git commit -m "Initial commit with Prisma Postgres integration"
+   git branch -M main
+   git remote add origin https://github.com/<your-username>/<repository-name>.git
+   git push -u origin main
+   ```
+   :::note
+    Replace `<your-username>` and `<repository-name>` with your GitHub username and the name of your repository.
+   :::
+2. Log in to [Vercel](https://vercel.com/) and navigate to your [Dashboard](https://vercel.com/docs/dashboard-features).
+3.  Create a new project. Follow Vercel's guide, but stop at the [step 3](https://vercel.com/docs/getting-started-with-vercel/import#optionally-configure-any-settings) where you will configure environment variables before clicking **Deploy**.
+4. Configure the `DATABASE_URL` environment variable:
+   1. Expand the **Environment variables** section.
+   2. Add the `DATABASE_URL` environment variable:
+     - **Key**: `DATABASE_URL`
+     - **Value**: Paste your Prisma Postgres `DATABASE_URL`.
```

Comment:
these are not indented for me, maybe try four spaces?

```suggestion
       - **Key**: `DATABASE_URL`
       - **Value**: Paste your Prisma Postgres connection URL, e.g. by copying it from the `.env` file in your Nuxt project.
```

![image](https://github.com/user-attachments/assets/d01157c5-a60f-495a-8926-551a2b918b89)